data/experiment-recipe-samples/rollout-mobile.json

```diff
@@ -0,0 +1,38 @@
+{
+  "schemaVersion": "0.1.0",
+  "id": "rollout-mobile-example",
+  "slug": "rollout-mobile-example",
+  "application": "reference_browser",
+  "appId": "org.mozilla.reference.browser",
+  "appName": "reference_browser",
```

Comment:
Should this be more specific? If we test both iOS and Android forms, we would need to have both Firefox versions.

Reply:
Good point @heres-maria maybe we should create a sample recipe for each of desktop/ios/android.